clippy.toml

```diff
@@ -7,6 +7,8 @@ disallowed-methods = [
 disallowed-types = [
     { path = "num_traits::AsPrimitive", reason = "Please use `From` or `TryFrom` with `OrderedFloat` instead." },
     { path = "num_traits::FromPrimitive", reason = "Please use `From` or `TryFrom` with `OrderedFloat` instead." },
+    { path = "num_traits::ToPrimitive", reason = "Please use `From` or `TryFrom` with `OrderedFloat` instead." },
```

Comment:
I'm not sure whether these clippy rules are applicable to traits. Have you tested it? 👀

Reply:
Yes. I enabled them before refactoring the code and they worked as expected. But I agree it may be more helpful if there are tests that expect these warnings. They can also be used to show the differences between these traits and From/TryFrom. Let me give this a try.

Reply:
Cannot `#[expect(clippy::disallowed_types)]` on `use` 🤦 https://rust-lang.github.io/rust-clippy/master/index.html#useless_attribute